### PR TITLE
[Tests-Only] Split webUiFiles suite into smaller suites

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
move some of the `webUIFiles` feature files into a separate suite folder
- webUIFiles
- webUIFilesActionMenu
- webUIFilesCopy
- webUIFilesDetails
- webUIFilesList
- webUIFilesSearch

These tests run in two different suites in CI `...Files1` and `...Files2`, 3 in each suite

## Related Issue
https://github.com/owncloud/web/issues/4876

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>FilesList</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13947/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13949/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13952/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13951/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13951/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13959/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13959/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/13959/

<p>




</p></details>

**Review by dpakach — CHANGES_REQUESTED** [review]
Having too many small suites will also hurt CI. It will occupy more drone agents to run tests while other builds will be pending. Furthermore, as the number of pipelines increases, we will just be doing the same mundane builds/setup tasks multiple times.
So I suggest you should divide it into fewer parts. 2 should be fine IMO.

File: .drone.star [hunk 1]
```diff
@@ -278,7 +299,18 @@ config = {
 				],
 				'webUIFavorites': 'OCISFavorites',
 				'webUIMarkdownEditor': 'OCISMarkdownEditor',
-				'webUIFiles': 'OCISFiles',
+				'OCISFiles1': [
```
Comment (kiranparajuli589) [inline .drone.star:302]:
```suggestion
				'webUIOCISFiles1': [
```

File: .drone.star [hunk 2]
```diff
@@ -51,7 +51,18 @@ config = {
...
+					'webUIFiles',
+					'webUIFilesActionMenu',
+					'webUIFilesCopy'
+
+				],
+				'webUIFiles2': [
+					'webUIFilesDetails',
+					'webUIFilesList',
+					'webUIFilesSearch'
+
```
Comment (dpakach) [inline .drone.star:64]: remove extra line

File: .drone.star [hunk 3]
```diff
@@ -278,7 +299,18 @@ config = {
...
+					'webUIFiles',
+					'webUIFilesActionMenu',
+					'webUIFilesCopy'
+
+				],
+				'OCISFiles2': [
+					'webUIFilesDetails',
+					'webUIFilesList',
+					'webUIFilesSearch'
+
```
Comment (dpakach) [inline .drone.star:312]: remove extra lines

**Review by dpakach — APPROVED** [review]
lgtm

**Review by individual-it — APPROVED** [review]

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14137/

<p>




</p></details>

**Pull request merged**